<launch>
	<node pkg="px4_path_planner" type="px4_path_planner_bspline_test" name="bspline" output="screen">
    </node>

</launch>
